lib/ansible/modules/net_tools/ldap/ldap_entry.py

```diff
@@ -145,9 +145,9 @@ def _load_attrs(self):
                 attrs[name] = []
 
             if isinstance(value, list):
-                attrs[name] = value
+                attrs[name] = [to_bytes(v) for v in value]
```

Comment:
Please could you use `map(to_bytes, value)` instead of the `for` loop?

Reply:
Done